### PR TITLE
Initial code for crictl

### PR DESCRIPTION
**Pull request by mrunalp**
@feiskyer PTAL

This is adding basic code for crictl. I will follow up with adding support for other commands.

### CODE REVIEW

File: cmd/crictl/main.go [hunk 1]
```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"time"
+
+	"github.com/Sirupsen/logrus"
```
Comment (feiskyer) [inline cmd/crictl/main.go]: replace with `github.com/golang/glog`? we used glog in other places.

File: cmd/crictl/version.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+	},
+}
+
+// Version sends a VersionRequest to the server, and parses the returned VersionResponse.
+func Version(client pb.RuntimeServiceClient, version string) error {
+	r, err := client.Version(context.Background(), &pb.VersionRequest{Version: version})
+	if err != nil {
+		return err
+	}
+	fmt.Printf("VersionResponse: Version: %s, RuntimeName: %s, RuntimeVersion: %s, RuntimeApiVersion: %s\n", r.Version, r.RuntimeName, r.RuntimeVersion, r.RuntimeApiVersion)
```
Comment (feiskyer) [inline cmd/crictl/version.go]: Add a `\n` for each ?

File: cmd/crictl/main.go [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+	"github.com/urfave/cli"
+	"google.golang.org/grpc"
+)
+
+const (
+	DefaultTimeout = 10 * time.Second
```
Comment (feiskyer) [inline cmd/crictl/main.go]: I think `DefaultTimeout` doesn't need to be exported, it also breaks validations because of no comments.

Comment (mrunalp) [inline cmd/crictl/main.go]: Fixed.

File: cmd/crictl/version.go (hunk 2 above)
Comment (mrunalp) [inline cmd/crictl/version.go]: Fixed.

File: cmd/crictl/main.go (hunk 1 above)
Comment (mrunalp) [inline cmd/crictl/main.go]: Fixed.

**Comment by feiskyer** [comment]
Cool. LGTM

**Pull request merged**